app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -217,6 +292,40 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
     if (filetype != None && filetype.get == "csv") {
       val file = getAccessScoreNeighborhoodsCSV(version = 2, coordinates)
       Future.successful(Ok.sendFile(content = file, onClose = () => file.delete()))
+    } else if(filetype != None && filetype.get == "shapefile"){
+      // Gather all of the data that will be written to the Shapefile.
```

Comment:
Can you move all this to a separate function in the same way we do for JSON and CSV?